packages/labs/react/src/create-component.ts

```diff
@@ -65,6 +65,8 @@ interface Options<I extends HTMLElement, E extends EventNames = {}> {
 
 type Constructor<T> = {new (): T};
 
+const MODE = window?.process?.env?.MODE;
```

Comment:
We can define `const DEV_MODE = true;` which will get replaced by rollup in the production build.

https://github.com/lit/lit/blob/a6a5591327a90e9c7d922d1d978627794909c422/rollup-common.js#L520

Example of how it's done for `reactive-element`:

https://github.com/lit/lit/blob/a6a5591327a90e9c7d922d1d978627794909c422/packages/reactive-element/src/reactive-element.ts#L39

and use it like
https://github.com/lit/lit/blob/a6a5591327a90e9c7d922d1d978627794909c422/packages/reactive-element/src/reactive-element.ts#L666-L675

So our production build wouldn't even contain code within `if (DEV_MODE) {}` with terser stripping away unused code blocks after replacing the const.

The labs/react package already has export conditions set for development and extends our common rollup plugin so there should be no extra config needed.

Reply:
nice! will do! thanks

Reply:
Done!